.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+name: CI
```

Comment:
If we are adding Github Actions I guess we need to remove CircleCI.

@santiagovidal from a business perspective, did RS migrate from CircleCI to Github Actions already?

Reply:
Yes, removing CircleCI is a pending task. I'll need some help with that:

- I don't have access to the settings page in this repo to remove the old CI and configure the new one. Can I be given access to configure it?

- What I can remove CircleCI configuration files from the repo. I guess that'll make the CircleCI check fail (which is a required check), so I guess we'll need to modify the rules before merging.

Reply:
Sorry for the late response, here goes:

Yes, we're ok with this change from a business perspective. We're currently using both, and it makes sense to (gradually) migrate and use a single provider.